.github/workflows/ci_cd.yml

```diff
@@ -1014,6 +1014,43 @@ jobs:
           issue-number: ${{ github.event.pull_request.number }}
           body: ${{ steps.help-body.outputs.help_body }}
 
+  lighthouse-ci:
+    name: Collect Lighthouse CI results
+    runs-on: ubuntu-latest
+    needs:
+      - nuxt-build
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup CI env
+        uses: ./.github/actions/setup-env
+        with:
+          setup_python: false
+          install_recipe: node-install
+
+      - name: Run build
```

Comment:
Could we reuse the built Nuxt app for this and for the k6?

Reply:
Building makes more sense, considering the time it takes

Reply:
Nuxt 3 made it a lot better :slightly_smiling_face: